### DISCUSSION TITLE
Indicate if a guide has already been shared in one of your own squads

### DISCUSSION THREAD

**Discussion by bobbyiliev** [dailydotdev/daily#1362] [Feature Requests]
I currently have a couple of squads, and I love using them! 👏

However, while browsing the feed, I sometimes end up sharing the same article in the same squad multiple times because I forget that I’ve already shared it a few days ago.

It would be really helpful if there was a small indicator in the article share UI to show that the article I'm about to share has already been shared in that specific squad.

Not a big issue but something that would be nice to have!

**Comment by idoshamun** [comment]
Hi @bobbyiliev,

I agree. We'll add to the backlog :)
